tgui/packages/tgui/interfaces/ListInputWindow/ListInputModal.tsx

```diff
@@ -95,6 +95,9 @@ export const ListInputModal = (props: ListInputModalProps) => {
   // |goonstation-change| send any text input to the search bar
   const onFocusSearch = (letter) => {
     let searchBarInput = getSearchBar();
```

Comment:
Not looked at surrounding code etc., but I *suspect* this can be made to use a `ref` instead. Not requiring for this PR, as out of scope. Your change is fine.

Reply:
ref would be great, but I don't think there's an easy way to get a tgui-core's input ref at the moment

![image](https://github.com/user-attachments/assets/84381338-ad84-4acd-8068-e2523fbb4145)